lib/rgb_matrix/engine.ex

```diff
@@ -61,27 +59,33 @@ defmodule RGBMatrix.Engine do
   end
 
   @doc """
-  Unregister a `RGBMatrix.Paintable` so the engine no longer paints pixels to it.
-  This function is idempotent.
+  Unregister a `RGBMatrix.Paintable` so the engine no longer paints pixels to
+  it. This function is idempotent.
   """
   @spec unregister_paintable(paintable :: module) :: :ok
   def unregister_paintable(paintable) do
     GenServer.call(__MODULE__, {:unregister_paintable, paintable})
   end
 
+  @spec interact(led :: LED.t()) :: :ok
+  def interact(led) do
```

Comment:
:information_source: (optional)

I mentioned an event system in one of my other comments. In that type of system this function could be something like this:

```ex
def send_event(event) do
```

```ex
@type t :: %Event{
  type: atom, # maybe
  source: any, # maybe - the thing that emitted the event
  x: float,
  y: float
}
```

Reply:
Maybe!  Let's do this in a future refactor.

Reply:
https://github.com/ElixirSeattle/xebow/issues/67